.esh/commands/reshape_i18n_keys.py

```
@@ -0,0 +1,394 @@
+		return None
+	with open(gitignore_path, "r", encoding="utf-8") as handle:
+		return pathspec.PathSpec.from_lines("gitwildmatch", handle)
+
+
+def is_i18n_rewrite_excluded(rel: str) -> bool:
+	return (
+		rel.startswith("src/public/locales/")
+		or rel.startswith("src/decl/")
+		or rel.startswith("src/scripts/checks/tools/")
+		or rel.startswith("tmp_")
+		or "/tmp_" in rel
```

Ruff (0.16.0)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)